### ISSUE TITLE
Sami is missing in the about us page

### ISSUE THREAD

**Issue by sarahmadeleine** [pledge4future/WePledge#182]

**Comment by sarahmadeleine** [comment]
Hey @sami1riaz do you want to add yourself here? :) https://github.com/pledge4future/WePledge/blob/about-us_update/frontend/pages/about-us.tsx

**Comment by sami1riaz** [comment]
@sarahmadeleine, yes, I'll add myself. Just need a decent picture of myself first lol
